tests/network-tests/src/scenarios/memberships.ts

```diff
@@ -24,4 +25,5 @@ scenario('Memberships', async ({ job }) => {
   job('transferring invites', transferringInvites)
   job('managing staking accounts', managingStakingAccounts)
   job('updating member verification status', updatingVerificationStatus).after(hireLeads)
+  job('updating member verify account', updateValidatorAccount).after(hireLeads)
```

Comment:
Please change the description to `updating member verify validator profile` also `updateValidatorAccount` should probably be renamed to `updateValidatorProfile`